pfl-basic/src/main/java17/org/glassfish/pfl/basic/reflection/Bridge.java

```diff
@@ -106,7 +106,8 @@ public Class<?> defineClass(Class<?> anchorClass, String className, byte[] class
         try {
             return getLookup(anchorClass).defineClass(classBytes);
         } catch (IllegalAccessException e) {
-            throw new RuntimeException("Unable to define class ", e);
+            System.out.println("Unable to define class " + className + " using anchor " + anchorClass + " due to " + e);
```

Comment:
I would not log it here, but I would use the message in the thrown exception:
```
throw new IllegalStateException("Unable to define class " + className + " using anchor " + anchorClass, e);
```
Also I prefer ISE to RTE, but it is not so important.